Update certs.go  | ADFS fix

MelroyB opened the pull request:
These added lines fix the ADFS issue as reported by multiple people.
You no longer receive the mitm error message when being redirected to the ADFS sub-domain

MelroyB commented:
1

Jemosj commented:
Hello, @MelroyB  really i am interested in your phishlets, can you please provide me with your ICQ or trillian or jabber, looking forward to deal with you.

Blussertje committed:
Update certs.go  | ADFS fix

These added lines fix the ADFS issue as reported by multiple people.
You no longer receive the mitm error message when being redirected to the ADFS sub-domain
